Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -0,0 +1,26 @@
+## How To Submit Content to EOS Documentation Portal
+
+**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
```

Comment:
This format breaks the style of the other documents to some degree. I think we should add here an overview of the actual submission process instead:

"Once you have your content written, you need to submit it for review. To that end, you need create a fork of the receiving repository and then add work on your own fork. From there, do a PR from your fork to the \`main\` branch of the target repo. DevRel then reviews the PR, suggests changes, if any, and after approval the PR can be merged."

Reply:
Done